janusgraph-hbase-parent/janusgraph-hbase-core/src/main/java/org/janusgraph/diskstorage/hbase/HBaseCompatLoader.java

```diff
@@ -24,9 +24,15 @@
 
     private static final Logger log = LoggerFactory.getLogger(HBaseCompatLoader.class);
 
-    private static final String DEFAULT_HBASE_COMPAT_VERSION = "1.0";
+    private static final String DEFAULT_HBASE_COMPAT_VERSION = "1.2";
 
-    private static final String DEFAULT_HBASE_CLASS_NAME = "org.janusgraph.diskstorage.hbase.HBaseCompat1_00";
+    private static final String HBASE_VERSION_1_STRING = "1.";
+
+    private static final String DEFAULT_HBASE_COMPAT_CLASS_NAME =
+        "org.janusgraph.diskstorage.hbase.HBaseCompat1_0";
+
+    private static final String[] HBASE_SUPPORTED_VERSIONS =
+        new String[] { "0.98", "1.0", "1.1", "1.2" };
```

Comment:
Yes, HBase 1.2 client works with 0.98 server.

But 1.2 and 0.98 have API differences. JanusGraph/Titan maintains two 'shims' to accommodate the differences.  We only bundle one version of hbase client jars in the dist.
Previously it was 0.98.  I changed it to 1.2.   The 1.2 client can work with the 0.98 cluster. But if the user drops in a set of 0.98 jars, it would work as well.   The 'shim' detects the version of the jars, and invokes accordingly.